src/features/rewards/tooltip/style.ts

```diff
@@ -6,23 +6,24 @@
 import styled from 'styled-components'
 
 interface StyleProps {
+  left?: string
```

Comment:
Can we instead of adding left prop, add `display: inline-block` to `StyledWrapper`. This way wrapper will take width of the child element and it will be always 50%?

Reply:
if this prop is not present it will default to `inline-block` at `50%` left. in the case of the site table, the tooltip with display as `inline-block` and at `50%` left is off, as it's pointing to the middle of the contribute table cell, and not the `x` button.

If not left, I think we should have some way to pass in a custom adjustment.